test/jdk/javax/swing/JPopupMenu/6691503/bug6691503.java

```diff
@@ -1,116 +0,0 @@
-/*
- * Copyright (c) 2008, Oracle and/or its affiliates. All rights reserved.
- * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
- *
- * This code is free software; you can redistribute it and/or modify it
- * under the terms of the GNU General Public License version 2 only, as
- * published by the Free Software Foundation.
- *
- * This code is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
- * version 2 for more details (a copy is included in the LICENSE file that
- * accompanied this code).
- *
- * You should have received a copy of the GNU General Public License version
- * 2 along with this work; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
- *
- * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
- * or visit www.oracle.com if you need additional information or have any
- * questions.
- */
-
-/*
- * @test
- * @key headful
- * @bug 6691503
- * @summary Checks that there is no opportunity for a malicious applet
- * to show a popup menu which has whole screen size.
- * a heaviweight popup menu is shown from an applet.
- * @author Mikhail Lapshin
- * @run main/othervm -Djava.security.manager=allow bug6691503
- */
-
-import javax.swing.*;
-import java.awt.*;
-
-public class bug6691503 {
-    private JPopupMenu popupMenu;
-    private JFrame frame;
-    private boolean isAlwaysOnTop1 = false;
-    private boolean isAlwaysOnTop2 = true;
-
-    public static void main(String[] args) {
-        bug6691503 test = new bug6691503();
-        test.setupUI();
-        test.testApplication();
-        test.testApplet();
```

Comment:
I guess we can only remove this `testApplet` as SM is invoked there only....we can still test `testApplication`, right?

Reply:
Updated. Repurposed the test.